src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -727,6 +727,7 @@ def write_string_to_logfile(self, message):
             cart_ctl.cfg_path.value = "."
             cart_ctl.m.value = message
             cart_ctl.n.value = None
+            cart_ctl.use_daos_agent_env.value = "--use_daos_agent_env"
```

Comment:
are we using dash or underscore?

Reply:
I think it'll be good to keep consistency at least for --use_daos_agent_env in this pr.

Reply:
With the change to using `use_daos_agent_env = FormattedParameter("--use_daos_agent_env", False)` the `.value` should be set to `True` to enable using the command line argument (https://github.com/daos-stack/daos/blob/master/src/tests/ftest/util/command_utils_base.py#L160-L161):
```suggestion
            cart_ctl.use_daos_agent_env.value = True
```
Sorry for not mentioning this explicitly in my previous review comment.